.golangci.yml

```diff
@@ -67,8 +67,6 @@ linters:
 linters-settings:
   depguard:
     include-go-root: true
-    packages-with-error-message:
-      - sync/atomic: "Use go.uber.org/atomic instead; see docs/dev/atomics.md"
```

Comment:
`sync/atomic` has clean APIs since 1.19 migration and we should not favor`uber/atomic` anymore.